test-infra/src/main/java/com/datastax/oss/driver/api/testinfra/session/SessionUtils.java

```diff
@@ -110,14 +125,26 @@ public static String getConfigPath() {
     return newSession(cassandraResource, keyspace, null, null, null, options);
   }
 
-  @SuppressWarnings({"unchecked", "TypeParameterUnusedInFormals"})
+  @SuppressWarnings("TypeParameterUnusedInFormals")
   public static <SessionT extends Session> SessionT newSession(
+      CassandraResourceRule cassandraResourceRule, DriverConfigLoaderBuilder loader) {
```

Comment:
This should take `DriverConfigLoader` instead of `DriverConfigLoaderBuilder`, will update.

Reply:
nevermind, found a way around it.